lib/dialects/postgres/query.js

```diff
@@ -303,7 +303,7 @@ class Query extends AbstractQuery {
         // there are multiple different formats of error messages for this error code
         // this regex should check at least two
         if (errDetail && (match = errDetail.replace(/"/g, '').match(/Key \((.*?)\)=\((.*?)\)/))) {
-          fields = _.zipObject(match[1].split(', '), match[2].split(', '));
+          fields = _.zipObject(match[1], match[2]);
```

Comment:
perhaps an old error style or copy and paste error?
I don't know enough about sequelize internals to know what to suggest?